src/render/tutorials/getting-started-with-docker/2_0.html.md

```diff
@@ -33,120 +33,164 @@ Project has the following structure:
-Open `knotx/conf/openapi.yml` and add following path definition:
+Let's take a look at `knotx/conf/openapi.yaml` file.
 
-```
-  /content/*:
+It's an [OpenAPI 3.0](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md) specification file that exposes APIs.
+
+> In OpenAPI terms, paths are endpoints (resources), such as /users or /reports/summary/, that your API exposes, and operations are the HTTP methods used to manipulate these paths, such as GET, POST or DELETE. [~source](https://swagger.io/docs/specification/paths-and-operations/))
+
+We can see three distinguishable parts of this file:
+
+ 1.  Information about our project, which is quite self-explanatory:
+ 
+      ```
+      ...
+      
+      openapi: "3.0.0"
+      info:
+        version: 1.0.0
+        title: Knot.x Starter Kit
+        description: Knotx Starter Kit
+        
+      ...
+      ```      
+ 2.  Server configuration that defines how our server behaves:
+ 
+      ```
+      ...
+      
+      servers:
+        - url: https://{domain}:{port}
+          description: The local API server
+          variables:
+            domain:
+              default: localhost
+              description: api domain
+            port:
+              enum:
+                - '8092'
+              default: '8092'
+      
+      ...
+      ```
+      Here we can see, for example, that our server will be registered on port `8092`.
+ 3.  Paths configuration that explicitly specifies our endpoints, their paths, methods, responses and much more in a clear, easily readable and declarative way:
+ 
+      ```
+      ...
+      
+      paths: 
+      
+        ...   
+             
+        /healthcheck:
+          get:
+            operationId: healthcheck-operation
+            # https://vertx.io/docs/vertx-health-check/java/
+            responses:
+              '200':
+                description: Everything is fine
+              '204':
+                description: No procedure are registered.
+              '501':
+                description: At least one procedure has reported a non-healthy stat
+              '500':
+                description: One procedure has thrown an error or has not reported a status in time
+      ...
+      ```
+      Our Starter Kit template comes with example paths and methods defined. Let's see the `/healthcheck` API. It is used to express the current state of our application in very simple terms: UP or DOWN.
+      
+      We can see that it is registered under `/healthcheck` path, it's method is a `GET` and it can have various responses depending on the state of the instance.
+      
+      There's no magic behind it. Healthcheck is handled by operation (more on that later in this tutorial) `healthcheck-operation` implemented as a handler in `HealthcheckHandlerFactory.java`. We will take a look at it later in this tutorial. Just to be brief: it simply calls our example endpoints (defined in `knotx/conf/openapi.yaml` as well). 
```

Comment:
> Just to be brief: it simply calls our example endpoints (defined in `knotx/conf/openapi.yaml` as well). 

I'm not really sure what you meant by this. Consider rephrasing it. Also, there's no obvious connection between `healthcheck-operation` id and `HealthcheckHandlerFactory` class at this point in the tutorial. Maybe you should be more clear that java classes are later assigned to the operation ids.

Reply:
fixed by moving this to final chapter